controllers/apply/building-connections/processor.js

```diff
@@ -1,5 +1,45 @@
 'use strict';
+const mail = require('../../../modules/mail');
 
 module.exports = function processor(formModel, formData) {
-    return Promise.resolve(formData);
+
+    const flatData = formModel.getStepValuesFlattened(formData);
+    const summary = formModel.getStepsWithValues(formData);
+
+    /**
+     * Construct a primary address (i.e. customer email)
+     */
+    const primaryAddress = `${flatData['first-name']} ${flatData['last-name']} <${flatData['email']}>`;
+    const organisationName = `${flatData['organisation-name']}`;
+
+    // @TODO determine an internal email address to send to for production environments
+
+    return mail.generateAndSend([
+        {
+            name: 'building_connections_customer',
+            sendTo: primaryAddress,
+            sendFrom: 'Big Lottery Fund <noreply@blf.digital>',
+            subject: 'Thank you for getting in touch with the Big Lottery Fund!',
+            templateName: 'emails/applicationSummary',
```

Comment:
Worth noting that this email doesn't include the short ID we generate, but until we firm up what the output will actually be here I don't think it's worth doing much more here.